README.md

```diff
@@ -2,18 +2,25 @@
   <img width="60px" src="https://user-images.githubusercontent.com/6180201/124313197-cc93f200-db70-11eb-864a-fc65765fc038.png" alt="giant microphone"/>   
  <br/>
   <h2 align="center">Whisper Playground</h2>
-  <h6 align="center">Instantly build speech2text apps in 99 languages using OpenAI's Whisper</h6>
-  <h6 align="center"> Live demo https://whisperui.monsterapi.ai</h6>
+  <h6 align="center">Instantly build real-time speech2text apps in 99 languages using OpenAI's Whisper, Diart, and Pyannote</h6>
```

Comment:
Mind reminding me to reach out to the Monster API folks once we merge? They might be able to deploy a new version of our repo